docs/rules/trials.md

```diff
@@ -0,0 +1,80 @@
+
+Throughout this whole process the presiding Judge is allowed to intervene in order to address procedural violations, keep order in the courtroom, or press on issues that need elaboration by the parties.
+
+###Objections
+
+Throughout the trial, both parties may raise objections as a formal form of protest to the Judge to disallow potentially unlawful witness testimony, or evidence which may be raised before the trial. As such objections will be broken down into 2 types: 
+
+**Witness Objections**: These are objections specific to a witness' testimony, or a party's conduct when dealing with a witness some common objection include:
+* Assuming Facts not in Evidence: The question makes an assumption that has not yet been proven.
+* Asked and Answered: The question has already been answered by the witness.
+* Badgering: The party asking the question is needlessly antagonizing the witness.
+* Compound Question: A question that contains two or more separate questions and requires multiple answers.
+* Hearsay: Testimony made about a statement made by someone other than the witness.
+* Inflammatory: The question is intended to cause the jury/judge to view a subject in an unreasonable prejudiced way. For example: asking a witness about supposed past crimes that never happened.
+* Non-Responsive Answer: The answer given does not respond to the question asked.
+* Leading Question: The question asked to the witness suggests its own answer. For example: "Is it true that you saw the defendant in the Engineering lobby". Such questions however may be asked in cross-examination or if the witness is openly antagonistic towards the party that called them to testify.
+* Relevance: The question is irrelevant to the case.
+* Vague Question: A question that is incomprehensible, incomplete, or leads to an ambiguous answer.
+* Foundation: The question relates to matters that the witness' personal experience does not extend to.
+* Speculation: The witness is asked to speculate or guess the answer.
+* Privilege: The witness may not legally answer a question. For example the captain cannot state information that has been classified by Nanotrasen.
+
+
+**Evidence objections**: These are objections specific to certain piece of evidence:
+*Best Evidence: The original source of evidence must be used as evidence when available. Otherwise evidence may be used if it can be authenticated for example a stamped copy of a document.
+*Fruit of the Poisonous tree: The evidence was obtained illegally.
+*Incomplete: The provided evidence is incomplete either because some of it was lost, or the party presenting it is withholding a part of it. For example a section of a document taken out of context.
+*Lack of Foundation: The evidence cannot be authenticated.
```

Comment:
Missing space before the text begins (i.e. `- Test` instead of `-Test`), Change list items to use `-` instead of `*`